### PR TITLE
Feature: Edit/removal of project updates

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

File: src/components/views/project/ProjectUpdates.tsx [hunk 1]
```diff
@@ -42,7 +49,86 @@ const ProjectUpdates = (props: { project?: IProject; fetchProject?: any }) => {
...
+			gToast(`Your update was deleted`, {
+				type: ToastType.SUCCESS,
+				// direction: ToastDirection.RIGHT,
+				title: 'Success!',
+				position: 'top-center',
+			});
+			return true;
+		} catch (error: any) {
+			console.log({ error });
+			return gToast(error?.message, {
```
Comment (RamRamez) [inline src/components/views/project/ProjectUpdates.tsx:123]: you can use this function for all toast errors:
`showToastError(err)`

File: src/components/views/project/ProjectUpdates.tsx [hunk 2]
```diff
@@ -103,14 +189,12 @@ const ProjectUpdates = (props: { project?: IProject; fetchProject?: any }) => {
 			props.fetchProject();
 			return gToast(`Your update was created`, {
 				type: ToastType.SUCCESS,
-				// direction: ToastDirection.RIGHT,
 				title: 'Success!',
 				position: 'top-center',
 			});
 		} catch (error: any) {
 			return gToast(error?.message, {
```
Comment (RamRamez) [inline src/components/views/project/ProjectUpdates.tsx:196]: you can use this function for all toast errors:
`showToastError(err)`

**Review by RamRamez — CHANGES_REQUESTED** [review]
Great! Thanks @mateodaza

File: public/images/trash.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (mateodaza) [reply public/images/trash.svg:1]: we're using it

**Review by RamRamez — APPROVED** [review]

**Pull request merged**